README.md

```diff
@@ -44,13 +44,13 @@ $(document).ready(function() {
 You can hook to the `play` and `pause` events like this:
 
 ```js
-$(window).on("quicktube:play", function(quicktubeId, $quicktubeEl) {
-    $("[data-show-while-playing]").show();
-});
+window.addEventListener('quicktube:play', () => {
+    showingItem.style.display = 'block';
+}, false);
 
-$(window).on("quicktube:pause", function(quicktubeId, $quicktubeEl) {
-    $("[data-show-while-playing]").hide();
-});
+window.addEventListener('quicktube:pause', () => {
+    showingItem.style.display = 'none';
+}, false);
```

Comment:
Have removed. We can extend this at a later date if we ever realise we need those events again